.github/workflows/update-analytics.yml

```diff
@@ -0,0 +1,34 @@
+name: Update Analytics Data
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Runs every day at midnight UTC
```

Comment:
Every day? would every week or every 2 weeks be sufficient? Or one a month considering that page is only showing monthly data?